tgui/packages/tgui/styles/components/Tabs.scss

```diff
@@ -1,57 +1,142 @@
+/**
```

Comment:
Unidentified regression in this file causing icon-only tabs to look weird

Reply:
Tabs.Tab now supports an `icon` property which should be used instead of <Icon> inside of a <Tabs.Tab>'s content.